packages/matchbox-css/tokens/font-size.scss

```diff
@@ -0,0 +1,26 @@
+@import '@sparkpost/design-tokens/dist/media-query.map.scss';
```

Comment:
This works for our existing maps, but the new style dictionary maps are all combined into one. Not sure how much work is involved there, but I think we can merge this in for now and address it later

Reply:
Yea I'll have to update to read from the new `tokens.scss` file from the updated tokens